front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -45,9 +51,94 @@ const ManchetteWithSpaceTimeChartWrapper = ({
 
   const [waypointsPanelIsOpen, setWaypointsPanelIsOpen] = useState(false);
 
+  // Cut the space time chart curves if the first or last waypoints are hidden
+  const spaceTimeChartLayersData = useMemo(() => {
+    let filteredProjectPathTrainResult = projectPathTrainResult;
+    let filteredConflicts = conflicts;
+
+    if (!waypointsPanelData) return { filteredProjectPathTrainResult, filteredConflicts };
```

Comment:
You can check here that there is more than 2 waypoints (remove the condition line 66)
```suggestion
    if (!waypointsPanelData || waypointsPanelData.filteredWaypoints.length <= 2) return { filteredProjectPathTrainResult, filteredConflicts };
```

Reply:
Done but with `< 2`